.github/workflows/README.md

```diff
@@ -0,0 +1,19 @@
+## Scheduled workflows
+
+| Action name                                    | Language   | Scheduled at                                                   | Function                     |
+|------------------------------------------------|------------|----------------------------------------------------------------|------------------------------|
+| [`ECDC.yml`](ECDC.yml)                         | R          | [12:00 every Thursday](https://crontab.guru/#0_12_*_*_4)       | Get recorded cases from ECDC |
+| [`JHU.yml`](JHU.yml)                           | Python     | [7:00 every day](https://crontab.guru/#0_7_*_*_*)              | Get recorded cases from JHU  |
+| [`LANL.yml`](LANL.yml)                         | R          | [0:00 every day](https://crontab.guru/#0_0_*_*_*)              |                              |
+| [`check-truth.yml`](check-truth.yml)           | Python & R | [13:00 every day](https://crontab.guru/#0_13_*_*_*)            |                              |
```

Comment:
Function: Check truth data (e.g., for negative incidence etc.); also update truth data report at https://github.com/epiforecasts/covid19-forecast-hub-europe/tree/main/data-truth